mpd/mpd.go

```diff
@@ -411,7 +411,7 @@ type SegmentTemplate struct {
 	AdaptationSet          *AdaptationSet   `xml:"-"`
 	SegmentTimeline        *SegmentTimeline `xml:"SegmentTimeline,omitempty"`
 	PresentationTimeOffset *uint64          `xml:"presentationTimeOffset,attr,omitempty"`
-	Duration               *int64           `xml:"duration,attr"`
+	Duration               *float64         `xml:"duration,attr"`
```

Comment:
Can you explain more the driver for this? Not only is it likely to break most implementations but doesn't look like it lines up with the DASH spec

Reply:
Yup, took a look and agree with you, the xsd types this as `xs:unsignedInt`. @flavioribeiro can you add more info on the background here?